emhttp/plugins/dynamix.vm.manager/VMSettings.page

```diff
@@ -162,8 +162,8 @@ _(Libvirt storage location)_:
 :vms_libvirt_volume_help:
 
 <?else: /* Libvirt is stopped */ ?>
-_(Libvirt vdisk size)_:
-: <input type="number" id="IMAGE_SIZE" name="IMAGE_SIZE" min="1" value="<?=htmlspecialchars($domain_cfg['IMAGE_SIZE']);?>" style="width:50px;" required="required" />_(GB)_ <span id="SIZE_ERROR" class="errortext"></span>
+_(Libvirt vdisk size)_ (_(GB)_):
```

Comment:
Splitting translations like this should be discouraged. Languages that are RTL or with different grammatical ordering for these sorts of things differently may feel odd with this sort of change.